src/RazorPageGenerator/Program.cs

```diff
@@ -4,8 +4,8 @@
 using System;
 using System.IO;
 using System.Linq;
-using Microsoft.AspNetCore.Razor;
-using Microsoft.AspNetCore.Razor.CodeGenerators;
+using System.Text;
+using Microsoft.AspNetCore.Razor.Evolution;
 
 namespace RazorPageGenerator
```

Comment:
Is it worth adding tests for `RazorPageGenerator`? I believe there are no tests for these right now. Not blocking this PR though.